### ISSUE TITLE
Coding Challenge : 1, 2, 5, 8, 13, 18 ....

### ISSUE THREAD

**Issue by shaikhanis** [Jay-study-nildana/TutoringTrainingStudents#58]
Coding challenge.

```
 Write a program to create following number series. 

 Print first n numbers.  Accept from user.              
1, 2, 5, 8, 13, 18, 25, 32
```

**Comment by Mayank-Jain-1** [comment]
Hi @shaikhanis here is a `Javascript` code snippet of a basic function that returns the above asked series. There is no such input in Javascript in the terminal but you can use `prompt` function which will work in the browser to take input. Or just you a input field.

```Javascript
const series = (n) => {
  if (n < 1) return []; 
  let answer = [1]
  for(var i = 1; i < n ; i++ ) {
     var number = answer[i-1] + Math.floor(i/2)*2 + 1 
     answer.push(number);
  }
  return answer;
}

var n = 15 // Length of the series to be printed
const answer = series(n)
console.log(answer);
````

**Comment by Jay-study-nildana** [comment]
@Mayank-Jain-1 

Thank you so much. I was able to put your code in codepen, and I can confirm it works. 

https://codepen.io/jay-pancodu/pen/BaPVzEW

```
// [object Array] (15)
[1,2,5,8,13,18,25,32,41,50,61,72,85,98,113]
```

I have no idea what the pattern in the number is. 

I literally have no clue how to do these puzzle things. When I was in college, I failed majority of coding tests and exams. and all the jobs I applied, they would give me coding tests, and I will just tell them that I am not eligible and walk out of the interview.

So, thank you for helping out here. much appreciated.
